src/screens/VisualTests/VisualTests.stories.tsx

```diff
@@ -359,6 +366,15 @@ export const Pending: Story = {
   },
 };
 
+export const NoPermission: Story = {
+  parameters: {
+    ...withBuilds({ storyBuild: pendingBuild, canReview: false }),
+    ...withFigmaDesign(
+      "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=2127-449276&mode=design&t=gIM40WT0324ynPQD-4"
+    ),
+  },
+};
```

Comment:
Do we need to add some stories for the other combinations of that guard (e.g. user can't review and build is not pending, or running build is in progress)?